src/DataFrames.jl

```diff
@@ -33,7 +34,7 @@ export AbstractDataFrame,
        deletecols!,
        deletecols,
        deleterows!,
-       describe,
+       DataAPI.describe,
```

Comment:
Is the prefix actually needed?

Reply:
`describe` is not exported by DataAPI.jl so I think it is needed.

Reply:
OK. It would be possible to do `using DataAPI: describe` above, but that's not clearly better.